src/containers/blocks/AddURLBlock.js

```diff
@@ -0,0 +1,83 @@
+import React, {Component} from 'react';
+import {connect} from "react-redux";
+import {bindActionCreators} from 'redux';
+import {addUrlToTest, hideUrlInput, toggleUrlInput, removeUrlFromTest} from "../../actions"
+
+class AddURLBlock extends Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      urlValue: ""
+    }
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.hideUrlInput();
+  }
+
+  handleAddUrl() {
+    this.props.toggleUrlInput();
+  }
+
+  handleInputChange(value) {
+    this.setState({
```

Comment:
It's generally suggested to use asynchronous setStates call

Reply:
Done.